doc/convert.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+
+for f in $(find $1 -name '*.adoc'); do
```

Comment:
Is this script still needed?

Reply:
I'll get rid of it when I ditch the adoc. I'm still not convinced we won't go back to it ;->

Reply:
Markdown is shit.

Reply:
There is always a backup plan to use LaTeX :)

Reply:
God yes.